### PR TITLE
.

### PR DESCRIPTION
**Pull request by Rwan19**

### CODE REVIEW

File: .idea/libraries/shape1 [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    val shapeName:String="" // variable 1
+    var shapeColor:String="" // variable 2
+
+    fun info(shapeName: String, shapeColor: String) { //function have to parameater
+        println("\nthe name of the shape is:$shapeName and the color of the shape is:$shapeColor")// print sentans
+
+    }
+
+    open fun computeArea(x:Double,y:Double):Double {
+        return x+y
```
Comment (shuruqt1000) [inline .idea/libraries/shape1:12]: what's are these for ? uncommented code

File: .idea/libraries/shape1 [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+//
+class Rectangle:shape() {  //supclass inhertans from superclass shape
+    val width=89 // variable 1
+    val height=78 // variable 2
+
+
+    override fun computeArea(width:Double,height:Double):Double { //function to calculate an area of Rectangle
+        val result=width*height
+        println(" the area of Rectangle is:")
```
Comment (shuruqt1000) [inline .idea/libraries/shape1:56]: uncommented

File: .idea/libraries/shape1 [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+//
+class Circle:shape() { //supclass inhertans from superclass shape
+    var pi:Double=3.14  //variable 1
+    var half:Double=10.0 // variable 2
+
+
+    override fun computeArea(pi:Double,radius :Double): Double { //function to calculate an area of Rectangle
+
+        var result:Double=(pi*(Math.pow(radius , 2.0)))
+        println(" the area of Circle is:$result")
```
Comment (shuruqt1000) [inline .idea/libraries/shape1:72]: uncommented

File: .idea/libraries/shape1 [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+//
+class Triangle : shape() { // supclass inhertans from superclass shape
+    val base=56 // variable 1
+    val height=67 // variable 2
+
+
+    override fun computeArea(base:Double,height:Double) :Double{ //function to calculate an area of triangle
+        val result=((base*height)/2)
+        println("the area of triangle is:")
+        return result
```
Comment (shuruqt1000) [inline .idea/libraries/shape1:41]: why do override explain you code in comment

**Comment by shuruqt1000** [comment]
see comment and fix

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]
please explain your code do not restate the sentences, I mean explain what values are passes or inherited why use open class in each part if use why is it important and how it affect the rest of the code

File: .idea/libraries/shape1 (hunk 1 above)
Comment (Rwan19) [reply .idea/libraries/shape1:12]: // fun to calculat the area of other class and return the area
  // i use open with function to i can use it in other class and modifcation on value

File: .idea/libraries/shape1 (hunk 2 above)
Comment (Rwan19) [reply .idea/libraries/shape1:56]: //function to calculate an area of triangle
 // the function is inhertied from fun computeAreain suberclass i use overrideto allwe to editt modifcation on value

Comment (Rwan19) [reply .idea/libraries/shape1:56]: by this equation
 Rectangle area = height times width  (A = h * w)

File: .idea/libraries/shape1 (hunk 3 above)
Comment (Rwan19) [reply .idea/libraries/shape1:72]: //i creat variable to store the value of the equation on it
// the function is inhertied from fun computeAreain suberclass i use overrideto hey ward allwe to editt modifcation on value

Comment (Rwan19) [reply .idea/libraries/shape1:72]: key ward

File: .idea/libraries/shape1 (hunk 4 above)
Comment (Rwan19) [reply .idea/libraries/shape1:41]: overrid is key ward
the function is inhertied from fun computeAreain in suberclass i use overrideto key ward allowed to editt modifcation on value.

**Comment by shuruqt1000** [comment]
good comments 
completed

**Pull request closed without merging**